### PR TITLE
Box - Apply container styles

### PR DESCRIPTION
**Pull request by novsstation**

### CODE REVIEW

**Review by EugeniyKiyashko — DISMISSED** [review]

File: js/renovation/ui/box/__tests__/box.test.tsx [hunk 1]
```diff
@@ -20,4 +26,68 @@ describe('Box > Attrs', () => {
...
+        const box = new Box(props);
+        let expectedDirection;
+        if (direction === 'row') {
+          expectedDirection = 'row';
+        } else if (direction === 'col') {
+          expectedDirection = 'column';
+        }
+
+        expect(box.cssStyles.flexDirection).toEqual(expectedDirection);
+      });
```
Comment (IgnatovDan) [inline js/renovation/ui/box/__tests__/box.test.tsx]: imho this is better than 'each':

      it('direction', () => {
        const props = {} as BoxProps;
        props.direction = 'row';
        expect(new Box(props).cssStyles.flexDirection).toEqual('row');

        props.direction = 'col';
        expect(new Box(props).cssStyles.flexDirection).toEqual('column');

        props.direction = null;
        expect(new Box(props).cssStyles.flexDirection).toEqual(null);

        props.direction = undefined;
        expect(new Box(props).cssStyles.flexDirection).toEqual(undefined);
      });

File: js/renovation/ui/box/__tests__/box.test.tsx [hunk 2]
```diff
@@ -20,4 +26,68 @@ describe('Box > Attrs', () => {
...
+  each([1, 'text', true, undefined, null]).describe('cssStyles: %o', (style) => {
+    it('Should pass styles to container from cssStyles', () => {
+      const box = mount<Box>(viewFunction({ cssStyles: style } as any));
+      const rootContainer = box.find(Widget);
+
+      expect(rootContainer.props().style).toEqual(style);
+    });
+  });
+
+  describe('cssStyles', () => {
```
Comment (IgnatovDan) [inline js/renovation/ui/box/__tests__/box.test.tsx]: These tests assign something to a props property and check the result.
Currently, the result is 'cssStyles' only, but there will be more results.
Let's rename it like this: describe('Check result of props values',

File: js/renovation/ui/box/__tests__/box.test.tsx [hunk 3]
```diff
@@ -20,4 +26,68 @@ describe('Box > Attrs', () => {
...
+
+        let justifyContent = align;
+        if (align === 'start') {
+          justifyContent = 'flex-start';
+        } else if (align === 'end') {
+          justifyContent = 'flex-end';
+        }
+
+        expect(box.cssStyles.justifyContent).toEqual(justifyContent);
+      });
```
Comment (IgnatovDan) [inline js/renovation/ui/box/__tests__/box.test.tsx]: imho this is better than 'each':

      it('align', () => {
        const props = {} as BoxProps;
        props.align = 'center';
        expect(new Box(props).cssStyles.justifyContent).toEqual('center');

        props.align = 'end';
        expect(new Box(props).cssStyles.justifyContent).toEqual('flex-end');

        props.align = 'space-around';
        expect(new Box(props).cssStyles.justifyContent).toEqual('space-around');

        props.align = 'space-between';
        expect(new Box(props).cssStyles.justifyContent).toEqual('space-between');

        props.align = 'start';
        expect(new Box(props).cssStyles.justifyContent).toEqual('start');

        props.align = null;
        expect(new Box(props).cssStyles.justifyContent).toEqual(null);

        props.align = undefined;
        expect(new Box(props).cssStyles.justifyContent).toEqual(undefined);
      });

File: js/renovation/ui/box/__tests__/box.test.tsx [hunk 4]
```diff
@@ -20,4 +26,68 @@ describe('Box > Attrs', () => {
...
+        let alignItems = crossAlign;
+        if (crossAlign === 'start') {
+          alignItems = 'flex-start';
+        } else if (crossAlign === 'end') {
+          alignItems = 'flex-end';
+        }
+
+        expect(box.cssStyles.alignItems).toEqual(alignItems);
+      });
+    });
```
Comment (IgnatovDan) [inline js/renovation/ui/box/__tests__/box.test.tsx:102]: imho this is better than 'each':

      it('crossAlign', () => {
        const props = {} as BoxProps;
        props.crossAlign = 'center';
        expect(new Box(props).cssStyles.alignItems).toEqual('center');

        props.crossAlign = 'end';
        expect(new Box(props).cssStyles.alignItems).toEqual('flex-end');

        props.crossAlign = 'start';
        expect(new Box(props).cssStyles.alignItems).toEqual('flex-start');

        props.crossAlign = 'stretch';
        expect(new Box(props).cssStyles.alignItems).toEqual('stretch');

        props.crossAlign = undefined;
        expect(new Box(props).cssStyles.alignItems).toEqual(undefined);

        props.crossAlign = null;
        expect(new Box(props).cssStyles.alignItems).toEqual(null);
      });

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/renovation/ui/box/__tests__/box.test.tsx (hunk 1 above)
Comment (novsstation) [reply js/renovation/ui/box/__tests__/box.test.tsx]: Yep, fixed

File: js/renovation/ui/box/__tests__/box.test.tsx (hunk 2 above)
Comment (novsstation) [reply js/renovation/ui/box/__tests__/box.test.tsx]: Yep, fixed

File: js/renovation/ui/box/__tests__/box.test.tsx (hunk 3 above)
Comment (novsstation) [reply js/renovation/ui/box/__tests__/box.test.tsx]: Yep, fixed

File: js/renovation/ui/box/__tests__/box.test.tsx (hunk 4 above)
Comment (novsstation) [reply js/renovation/ui/box/__tests__/box.test.tsx:102]: Yep, fixed

**Review by KuznetsovVN — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**